pages/index.vue

```diff
@@ -39,11 +79,56 @@ export default class extends Vue {
     [0, 0, 0, 0, 0, 0, 0, 0]
   ]
 
-  message: string = 'あなたの番です'
+  // オセロ用
+  message = 'あなたの番です'
+
+  // メモ機能用
+  count = 0
+  memoList: Array<Memo> = []
+  memo = ''
+
+  // 計算機用
+  calculatorAnswer = 0
+  numA = 0
+  numB = 0
+  numTool = ''
 
+  // オセロ石用クリックイベント
   public onClick(x: number, y: number) {
     alert(x + 'と' + y)
   }
+
+  // メモ機能用クリックイベント
+  public onMemoClick(memo: String, memoDone: Boolean) {
+    const registMemo = {
+      id: this.count++,
+      value: memo,
+      done: memoDone
+    }
+    this.memoList.push(registMemo)
+  }
+  public onDoneClick(id: number) {
+    this.memoList[id].done = true
+  }
+
+  // 計算機能用クリックイベント
+  public onAnswerClick(numA: number, numB: number, numTool: String) {
+    alert(numTool)
+    switch (numTool) {
+      case '+':
+        this.calculatorAnswer = numA + numB
+        break
+      case '-':
+        this.calculatorAnswer = numA - numB
+        break
+      case '×':
+        this.calculatorAnswer = numA * numB
+        break
+      case '÷':
+        this.calculatorAnswer = numA / numB
+        break
+    }
+  }
```

Comment:
this.calculatorAnswer = this.operators.filter(operator => operator.id === this.selectedOperatorId)[0]).culc(numA, numB)